nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -47,20 +47,21 @@ def _ml_parametrization(
         alpha: float,
         learning_rate: str,
         regressor: str,  # Choice of learner.
-        noise_free: bool  # Whether we work on the test set (the real cost) on an approximation (CV error on train).
+        noise_free: bool  # Whether we work on the test set (the real cost) on an approximation (CV error on train). -> not really noise
```

Comment:
@teytaud the name is really confusing, since the function remains deterministic once instantiated